test/quicapitest.c

```diff
@@ -2658,6 +2658,39 @@ static int test_ssl_new_from_listener(void)
 
 OPT_TEST_DECLARE_USAGE("provider config certsdir datadir\n")
```

Comment:
This line (and the comment above it) should be below all the tests (i.e. just before the declaration of `setup_tests`)